preset/rabbitmq/preset_test.go

```diff
@@ -105,3 +105,119 @@ func TestPreset_withManagement(t *testing.T) {
 
 	require.Equal(t, http.StatusUnauthorized, resp.StatusCode)
 }
+
+func TestPreset_withMessages(t *testing.T) {
+	t.Parallel()
+
+	messages := []rabbitmq.Message{
+		{
+			Queue:       "events",
+			ContentType: "text/plain",
+			StringBody:  "order: 1",
+		},
+		{
+			Queue:       "alerts",
+			ContentType: "text/plain",
+			StringBody:  "CPU: 92",
+		},
+	}
```

Comment:
WDYT about merging `withMessages` and `withBinaryMessages` tests into a single test, and moving them both into `TestPreset` flow? Management test is a separate one just because it uses a different docker image, so it has to be a separate docker container.

Reply:
It would be fine by me however, as I mentioned in one of previous responses, linter for some reason throws an error if there is a function that's longer than 60 loc which would be the case if the tests were merged. You can see that in [`5fecd7c` (#66)](https://github.com/orlangure/gnomock/pull/66/commits/5fecd7c7b7e30e02005855c2844638f91daaf328).

Reply:
Regarding `funlen` linter in tests, that's fine if it makes sense. I think here it makes a lot of sense to be a single test. It runs faster (only 3 container is created and not 3) and tests all the supported features together, instead of testing only one each time.